lib/v2/screens/import_key/import_key_screen.dart

```diff
@@ -25,7 +27,7 @@ class _ImportKeyScreenState extends State<ImportKeyScreen> {
   @override
   void initState() {
     super.initState();
-    _importKeyBloc = ImportKeyBloc();
+    _importKeyBloc = ImportKeyBloc(SettingsNotifier.of(context), AuthNotifier.of(context));
```

Comment:
We are going to keep the Auth notifier. It was a lot of work to remove it and create a new architecture.

Reply:
Ok I understand the Auth notifier, but the Settings notifier is now the singleton, is the value retrieved on another screen that does not use the singleton?